src/pa01/snakes_and_ladders.py

```diff
@@ -0,0 +1,141 @@
+import random
```

Comment:
NOTE: Missing names and email!  Add this to every script. I recommend setting it as a template in pycharm so that you get it automatically whenever you create a new `.py` file.